test/RoslynAnalyzer.Tests/RoslynAnalyzer.Tests.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Nullable>enable</Nullable>
+    <LangVersion>Preview</LangVersion>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(MonoBuild)' == ''">
+    <TargetFramework>net5.0</TargetFramework>
+    <DefineConstants>$(DefineConstants);ILLINK</DefineConstants>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(MonoBuild)' != ''">
+    <TargetFramework>net472</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Analyzer.Testing.XUnit" Version="1.0.1-beta1.*" />
+     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.7.0" />
+     <PackageReference Include="Microsoft.CodeAnalysis.VisualBasic.Analyzer.Testing.XUnit" Version="1.0.1-beta1.*" />
```

Comment:
What is this for?

Reply:
This analyzer should be functional in VB as well, but I decided not to explicitly test it because it doesn't seem worth the effort.